docs/source/modules/plugins.rst

```diff
@@ -0,0 +1,106 @@
+
```

Comment:
so will this file be auto-generated by the readthedocs build? I.e. won't be part of the repository once that happens?